cloudera/tests/test_unit.py

```diff
@@ -96,61 +93,6 @@ def test_given_cloudera_check_when_not_supported_version_then_emits_critical_ser
     )
 
 
-def test_given_cloudera_check_when_supported_version_then_emits_ok_service(
```

Comment:
To reviewer: I removed this test because the assertions of this test are included in `test_given_cloudera_check_when_good_health_cluster_then_emits_cluster_metrics`.